server/game/cards/05_LOF/units/BaylanSkollEnigmaticMaster.ts

```diff
@@ -0,0 +1,41 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardCardType } from '../../../core/Constants';
+import { CostAdjustType } from '../../../core/cost/CostAdjuster';
+import * as EnumHelpers from '../../../core/utils/EnumHelpers';
+
+export default class BaylanSkollEnigmaticMaster extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: 'baylan-skoll#enigmatic-master-id',
+            internalName: 'baylan-skoll#enigmatic-master',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'Use the Force',
```

Comment:
We have to add why we use the force in title for disambiguation of multiple trigger